mk/linux/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:bionic
+
+ENV SHELL=/bin/bash
```

Comment:
Required for the 0ad spidermonkey build; it's unlikely this would be needed here.